Resources/Prototypes/Entities/Mobs/Customization/Markings/human_hair.yml

```
@@ -1188,6 +1188,13 @@
   sprites:
     - sprite: Mobs/Customization/human_hair.rsi
       state: longest2
+      - type: marking
+  id: HumanHairVeryshortovereye
```

```suggestion
  id: HumanHairVeryShortOverEye
```
(don't forget to fix the loc strings too)